force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -83,8 +98,14 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the number is even, False otherwise.
      */
     public static Boolean isEven(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+       Integer remainder = Math.mod (num, 2);
+       if (remainder==0) {
+        return true;
+       }
+        else {
+        return false; // Replace null with the variable you used to store the result
```

Comment:
Since this method returns a Boolean, for practice see if you can remove the if statement and do this in one line